src/HasModels.js

```diff
@@ -19,12 +24,12 @@ export default class HasModels {
 
   /**
    * Override to define the model names to access
-   * @return {object} (model identifier: class attribute) pairs
-   * @example model_names() { 
-   * return {'user': 'User'}
+   * @return {array|object} Model identities to add to this.models, or object of {identity: name}
```

Comment:
Will update the example to just use a string, either is acceptable now (if you want to provide a short/alternate name you can return a mapping, normal would be array of strings)